spec/fluent.ebnf

```diff
@@ -3,13 +3,24 @@
 Resource            ::= (blank_line | Entry | junk_line)*
 Entry               ::= Message
                       | Term
-                      | (ResourceComment | GroupComment | Comment)
-Message             ::= Comment? Identifier inline_space? "=" inline_space? ((Pattern Attribute*) | (Attribute+)) line_end
+                      | Section
+                      | ValueLessMessage
+                      | (ResourceComment | GroupComment | Comment | LegacyComment)
+Message             ::= (Comment | LegacyComment)? Identifier inline_space? "=" inline_space? ((Pattern Attribute*) | (Attribute+)) line_end
 Term                ::= Comment? TermIdentifier inline_space? "=" inline_space? Value Attribute* line_end
```

Comment:
You're not adding `| LegacyComment` here because Terms weren't around pre-0.5? Nice.